.gitignore

```diff
@@ -65,4 +65,6 @@ src/pluginMap.ts
 /apps_backup/*
 
 # Federation Temp files
-/.__mf__temp
\ No newline at end of file
+/.__mf__temp
+src/components/Licenses/beBomData.json
+src/components/Licenses/feBomData.json
```

Comment:
let's keep this inside the `public` dir. instead. 

(we'd also be able to access it by `care.ohc.network/licenses/backend|frontend.json`)